src/views/HomeView.vue

```diff
@@ -245,9 +251,14 @@ p {
   margin: 0 auto;
 }
 
-.create-game-btn {
-  display: flex;
-  justify-content: flex-end;
+.home-card-games {
+  padding: 0;
+  max-width: 640px;
```

Comment:
I think this is a bit small for large devices. There's a fair bit of empty space here

![image](https://github.com/cuttle-cards/cuttle/assets/6520704/b6cc4dec-58c0-4f9e-9a49-2916adb23afc)